File: MiracleMileAPI/MiracleMileAPI/Model/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MiracleMileAPI.Model
{
    public class Address
    {
        public string City { get; set; }
        public string Country { get; set; }
        public string Street { get; set; }
        public string ZipCode { get; set; }
        public string Attention { get; set; }
        public string CareOf { get; set; }
    }
}

File: TopGun/src/app/model/address.ts
export class Address {
    public city: string;
    public country: string;
    public street: string;
    public zipCode: string;
    public attention: string;
    public careOf: string;

    constructor(data: Partial<Address>) {
        Object.assign(this, data);
    }

    public static empty(): Address {
        return new Address({
            city: '',
            country: '',
            street: '',
            zipCode: '',
            attention: '',
            careOf: ''
        });
    }

}
